vignettes/sign-commits.Rmd

```diff
@@ -0,0 +1,166 @@
+---
+title: "Sign commits (GPG)"
+author: "Ildiko Czeller"
+date: "`r Sys.Date()`"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Sign commits (GPG)}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+
+It is good practice to sign your commits so that your commits can be verified as certainly made by you and not by someone impersonating you.
+
+One technology to enable this is [GPG](https://gnupg.org/) for which and R interface is created by [Jeroen Ooms](https://github.com/jeroen). The current package provides helper functions and aim to ease the process as much as possible.
```

Comment:
Might be helpful to link to Jeroen's package (https://github.com/jeroen/gpg) rather than just his github. I also might rephrase this sentence to something like "One technology to enable this is GPG, which has an R wrapper written by Jeroen Ooms."